dashboard2/settings/jptt.md

```diff
@@ -151,3 +151,12 @@ HTML:
 "%"."exampleObject"."exampleNestedHTMLinJS"."nestedHTML" html
 ```
 **Note:** The JSON Path tester tool is **not equipped** to display the nested use case.
+
+#### Skip
+
+Use this processing mode to mark child node not to be translated. Order is important, skip rules must come after the generic * rule. The exclamation mark in the end of the rule is mandatory, so please use the following syntax: `! skip`
```

Comment:
Is this true, though? Doesn't the `!` actually _disable_ the rule in question without removing it?